packages/mock-block-dock/src/debug-view/header.tsx

```diff
@@ -134,18 +134,19 @@ export const Header = ({ darkMode, setDarkMode }: Props) => {
         </Typography>
       </Link>
 
-      <Button onClick={() => setDarkMode((prev) => !prev)} sx={{ mr: 1 }}>
-        Dark Mode
-        {darkMode ? (
-          <OnSwitch sx={{ height: 20, width: 40, ml: 1.25 }} />
-        ) : (
-          <OffSwitch sx={{ height: 20, width: 40, ml: 1.25 }} />
-        )}
-      </Button>
+      {/* @todo restore when styling service is implemented */}
+      {/* <Button onClick={() => setDarkMode((prev) => !prev)} sx={{ mr: 1 }}> */}
+      {/*  Dark Mode */}
+      {/*  {darkMode ? ( */}
+      {/*    <OnSwitch sx={{ height: 20, width: 40, ml: 1.25 }} /> */}
+      {/*  ) : ( */}
+      {/*    <OffSwitch sx={{ height: 20, width: 40, ml: 1.25 }} /> */}
+      {/*  )} */}
+      {/* </Button> */}
 
       <Button onClick={() => setDebugMode(false)}>
         Preview Mode
-        <OnSwitch sx={{ height: 20, width: 40, ml: 1.25 }} />
+        <OffSwitch sx={{ height: 20, width: 40, ml: 1.25 }} />
```

Comment:
If we stick with this UI for longer, might be nice to replace this with the MUI toggle component (which [can be customised](https://mui.com/material-ui/react-switch/#customization) to match the styling here, but comes with animations out of the box which would help make this feel less static). But we should probably address this down the line, may be worth adding a todo here though (if you agree).

Reply:
Done in https://github.com/blockprotocol/blockprotocol/pull/622/commits/3315ddd5dfe7948d739ca4a67ffa90e933e7d1da